Source/missiles.cpp

```diff
@@ -957,39 +922,15 @@ bool MonsterTrapHit(int m, int mindam, int maxdam, int dist, missile_id t, bool
 {
 	auto &monster = Monsters[m];
 
-	bool resist = false;
-	if (monster.mtalkmsg != TEXT_NONE) {
+	bool resist = monster.IsResistant(t);

```

Comment:
Move to 944 since it's not used until then.

Reply:
OK. I did't change this function much, because I change it a lot in main PR. In this split this line only hooks to function moved to monster.cpp.